### ISSUE TITLE
How to use vueuse/head?

### ISSUE THREAD

**Issue by thalesagapito** [frandiox/vite-ssr#6]
Since the renderToHtml call is inside the module, would it be possible to install and use https://github.com/vueuse/head?

**Comment by frandiox** [comment]
@thalesagapito I'm integrating vueuse/head in a local branch already and will probably merge it this week. For now, there's a component called `Helmet` to do the same thing. Example [here](https://github.com/frandiox/vite-ssr/blob/master/examples/vue/src/pages/Homepage.vue#L2).

**Comment by thalesagapito** [comment]
Cool! Nice work, I'll close this for now then, and wait for the integration

**Comment by frandiox** [comment]
@thalesagapito Released in `0.5.0`!

```js
import { createHead } from '@vueuse/head'

export default viteSSR(App, { routes }, ({ app }) => {
  const head = createHead()
  app.use(head)

  return { head }
})
```
